signin.css

```
@@ -0,0 +1,137 @@
+* {
+    box-sizing: border-box;
+  }
+  
+  html {
+    word-break: keep-all;
+  }
+  
+  body {
+    margin:50px auto;
+  }
+
+
+  header{
+    display:flex;
+    flex-direction: column;
+    align-items: center;
+    flex-shrink: 0;
+  }
+
+  label {
+    color: #2c2c2c;
+    display: block;
+    color: var(--Secondary-800, #1F2937);
+    font-family: Pretendard;
+    font-size: 18px;
+    font-style: normal;
+    font-weight: 700;
+    line-height: 26px;
+    margin: 16px 0 8px;
+  }
+
+  button {
+    border-radius: 40px;
+    background: var(--Primary-100, #3692FF);
+    border: none;
+    display: block;
+    height: 56px;
+    padding: 16px 124px;
+    justify-content: center;
+    align-items: center;
+    gap: 10px;
+    width: 100%;
+  }
+
+  input {
+    display:block;
+    margin: 24px 0;
+    padding: 16px 24px;
+    background-color: #f3f4f6;
+    border: none;
+    border-radius: 12px;
+    font-size: 16px;
+    line-height: 24px;
+    width: 100%;
+   }
+  
+  footer{
+    margin:0 0;
+  }
+  /*판다로고*/
+  #panda{
+    width: 396px;
+    height: 132px;
+  }
+
+  .bigLogo{
+    display: flex;
+    width: 100%;
+    margin: 0 auto;
+    justify-content: center;
+    align-items: center;
+  }
+
+  /*아래쪽 전체 틀*/
+  .login {
+    margin: 40px auto 0 auto;
+    width:60%;
+  }
+
+  /*아이콘 틀(간편로그인)*/
+  .linklog{
+    background-color:#E6F2FF;
+    width:100%;
+    margin:24px 0;
+    border-radius: 8px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 16px 23px;
+    align-self: stretch;
+   }
+
+  /*아이콘*/
+
+  .icons{
+  display:flex;
+  gap:10px;
+  }
```

P3:
가독성을 위해 indent를 맞춰주시면 좋을 것 같습니다.

```suggestion
  .icons{
    display:flex;
    gap:10px;
  }
```